src/app/carbon-estimation/carbon-estimation.component.css

```diff
@@ -16,25 +16,29 @@
 }
 
 ::ng-deep .web-logo {
-  background-image: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 -960 960 960'%3E%3Cpath style='fill:white' d='M160-160q-33 0-56.5-23.5T80-240v-480q0-33 23.5-56.5T160-800h640q33 0 56.5 23.5T880-720v480q0 33-23.5 56.5T800-160H160Zm0-80h420v-140H160v140Zm500 0h140v-360H660v360ZM160-460h420v-140H160v140Z'/%3E%3C/svg%3E");
+  background-image: url('../../assets/web.svg');
```

Comment:
I moved the SVGs to separate files because Angular was throwing an error due to the size of this CSS file when building the app. (There is also the option to increase the allowed file size in the angular.json file.)

Reply:
Just had a quick check and this seems to work as expected. I think initially we were concerned that the constantly changing tooltip would make a request for the image each time. Also looks to include the images when creating the npm package.